src/core/net/tcp6.hpp

```diff
@@ -289,14 +301,15 @@ class Tcp : public InstanceLocator, private NonCopyable
          *
          * @retval kErrorNone    Successfully aborted the TCP endpoint's connection.
          * @retval kErrorFailed  Failed to abort the TCP endpoint's connection.
+         *
          */
         Error Abort(void);
 
         /**
          * Deinitializes this TCP endpoint.
          *
          * This means that OpenThread no longer keeps track of this TCP endpoint and
-         * deallocates all resources it has internally allocated for this TCP endpoint.
+         * de-allocates all resources it has internally allocated for this TCP endpoint.
```

Comment:
I believe `deallocates` is accepted as a word: https://www.google.com/search?q=deallocates
```suggestion
         * deallocates all resources it has internally allocated for this TCP endpoint.
```